### PR TITLE
Beautify changes

### PR DESCRIPTION
**Pull request by revant**
> Please provide enough information so that others can review your pull request:

> Explain the **details** for making this change. What existing problem does the pull request solve?

TODO

- add changes that make helm charts work

### CODE REVIEW

File: build/worker/Dockerfile [hunk 1]
```diff
@@ -85,7 +85,7 @@ RUN apt-get update \
     xvfb \
     libfontconfig \
     wkhtmltopdf \
-    # For healthcheck.sh in helm chart
+    # For healthcheck
     wait-for-it \
```
[the remark this answers is not in the document]
Comment (revant) [reply build/worker/Dockerfile:89]: wait-for-it is required. I'm using it directly instead of using it through healthcheck.sh wrapper.

**Comment by revant** [comment]
- refactor missed a feature sent by PR https://github.com/frappe/frappe_docker/pull/441
- removed healthcheck.sh script wrapper. Using wait-for-it directly.
- added gevent_patch.py

With this I am able to refactor helm-charts and make them work. I'll write migration guide as well.

File: build/worker/Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (vrslev) [reply build/worker/Dockerfile:89]: But you removed `healthcheck.sh` from Dockerfile 🤔

[the remark this answers is not in the document]
Comment (vrslev) [reply build/worker/Dockerfile:89]: Where are you using wait-for-it?

[the remark this answers is not in the document]
Comment (vrslev) [reply build/worker/Dockerfile:89]: Never mind, found it in your Helm chart fork.

**Comment by vrslev** [comment]
I added `healthcheck.sh` _only_ for tests. Merging!

**Pull request merged**